wgsl/index.bs

```diff
@@ -9182,7 +9182,7 @@ Each is described in detail in subsequent sections.
 <table class='data'>
   <caption>Built-in input and output values</caption>
   <thead>
-    <tr><th>Name<th>Stage<th>Direction<th>Type
+    <tr><th>Name<th>Stage<th>Direction<th>Type<th>[=Extension=]
```

Comment:
nit: Every row should have a corresponding `<td>` so that the styling extends across the whole row.

Otherwise LGTM but a WGSL editor should review :)